### DISCUSSION TITLE
SO Only detect DHCP and DNS

### DISCUSSION THREAD

**Discussion by AlxSIO** [Security-Onion-Solutions/securityonion#8083] [Unsupported Versions] [answered]
Greetings,
I've managed to install and use SO but im having this issue where SO Only detect DHCP and DNS but the guides I followed showed much more (like http ... ) Is there a way to fix this ?

**Comment by dougburks** [comment, marked as answer]
This most likely means that your sniffing interface is only seeing partial traffic, perhaps only broadcast traffic. Check your tap or span port to make sure it is sending all the traffic that you want to monitor.

https://docs.securityonion.net/en/2.3/hardware.html#packets
